test/unitTest/java/org/thoughtcrime/securesms/PinnedMessagesMocks.java

```diff
@@ -0,0 +1,82 @@
+package org.thoughtcrime.securesms;
+
+import android.content.Context;
+
+import org.junit.runner.RunWith;
+import org.mockito.BDDMockito;
+
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+import org.thoughtcrime.securesms.database.DatabaseFactory;
+import org.thoughtcrime.securesms.database.MessagingDatabase;
+import org.thoughtcrime.securesms.database.model.MessageRecord;
+import org.thoughtcrime.securesms.database.MmsDatabase;
+import org.thoughtcrime.securesms.database.MmsSmsDatabase;
+import org.thoughtcrime.securesms.database.SmsDatabase;
+
+import static org.mockito.Mockito.when;
+
+
+/**
+ * Created by BABO99 on 2018-02-09.
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({DatabaseFactory.class})
+public class PinnedMessagesMocks extends BaseUnitTest {
```

Comment:
is it a common pattern to have mocks be in the superclass ?

Reply:
@g-harel  yes, it helps to separate the setups from the actual tests